pyzm/ml/object.py

```diff
@@ -28,12 +28,15 @@ def __init__(self, options={}):
         if self.options.get('object_framework') == 'opencv':
             import pyzm.ml.yolo as yolo
             self.model =  yolo.Yolo(options=options)
-            
 
         elif self.options.get('object_framework') == 'coral_edgetpu':
             import pyzm.ml.coral_edgetpu as tpu
             self.model = tpu.Tpu(options=options)
 
+        elif self.options.get('object_framework') == 'aws_rekognition':
+            import pyzm.ml.aws_rekognition as awsr
```

Comment:
Let's put this import in a try catch. If the module is missing, generate an error using the logger object asking folks to install aws_rekognition (reason coming up later)

Reply:
Done.